Client/Frontend/Widgets/ErrorToast.swift

```diff
@@ -7,15 +7,15 @@ import SnapKit
 
 private struct ErrorToastDefaultUX {
     static let cornerRadius: CGFloat = 40
-    static let fillColor = UIColor.Photon.Red70
     static let margins = UIEdgeInsets(top: 10, left: 12, bottom: 10, right: 12)
-    static let textColor = UIColor.Photon.White100
+    static let textColor = UIColor(rgb: 0xBD1531)
```

Comment:
Can use `red300` from rewardsUI.Constants

Reply:
Those colors aren't currently public, it'd probably be better to extract that pallet out of BraveRewardsUI but maybe not in scope for this PR

Reply:
Works for me